packages/manager-react-components/src/components/tiles/BillingInformationsTile/BillingInformationsTile.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { BillingInformationsTile } from './BillingInformationsTile';
+import { BillingInformationsDecorator } from './utils';
+
+export const BillingInformationsTileDefault = () => (
+  <BillingInformationsTile resourceName="my-resource-name">
+    <BillingInformationsTile.Title>Subscription</BillingInformationsTile.Title>
+    <BillingInformationsTile.Divider />
+    <BillingInformationsTile.CreationDate />
+    <BillingInformationsTile.Divider />
+    <BillingInformationsTile.State />
+    <BillingInformationsTile.Divider />
+    <BillingInformationsTile.NextBillingDate />
+    <BillingInformationsTile.Divider />
+    <BillingInformationsTile.ResiliateLink />
+  </BillingInformationsTile>
+);
```

Comment:
With this syntax you have to repeat the Divider component every time.
Do we really need the flexibility that comes with this syntax ?
Most of the Billing tiles look alike (and that's a good thing for UX I think) and the elements are ordered the same.